libdevcore/Exceptions.h

```diff
@@ -49,6 +49,7 @@ DEV_SIMPLE_EXCEPTION(BadHexCharacter);
 DEV_SIMPLE_EXCEPTION(BadHexCase);
 DEV_SIMPLE_EXCEPTION(FileError);
 DEV_SIMPLE_EXCEPTION(DataTooLong);
```

Comment:
Why not use `DataTooLong`?

Reply:
Thought about it, but it's a very different thing.